baseapp/state.go

```diff
@@ -6,8 +6,12 @@ import (
 )
 
 type state struct {
-	ms           sdk.CacheMultiStore
-	ctx          sdk.Context
+	ms  sdk.CacheMultiStore
+	ctx sdk.Context
+	// eventHistory accumulates events returned by DeliverTx throughout a block.
+	// The accumulated events are passed to the EndBlocker in its context's
+	// EventManager ABCI event history, and the state's eventHistory is then cleared.
+	// Not used for modes or states other than delivery.
 	eventHistory []abci.Event
```

Comment:
```suggestion
	// eventHistory accumulates events returned by DeliverTx throughout a block.
	// [AGORIC] The accumulated events are passed to the EndBlocker in its context's
	// EventManager ABCI event history, and the state's eventHistory is then cleared.
	// Not used for modes or states other than delivery.
	eventHistory []abci.Event
```

Reply:
Done - and that's a fine way to resolve our tagging needs vs godoc's requirements.